### PR TITLE
feat: add serve & integration to README

### PR DESCRIPTION
**Pull request by matthisholleville**
Closes # 

## 📑 Description

Add `serve` & `integrations` documentation to README.

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,6 +272,44 @@ The Kubernetes system is trying to scale a StatefulSet named fake-deployment usi
 
 </details>
 
+_Manage integrations_
```
Comment (AlexsJones) [inline README.md:275]: Can we add the whole block in details? I think we're getting to a point where ALL of these commands should be in docs.k8sgpt.ai only as the README is getting very long, what do you think?

Comment (matthisholleville) [reply README.md:275]: I agree

File: README.md [hunk 2]
```diff
@@ -269,6 +272,48 @@ The Kubernetes system is trying to scale a StatefulSet named fake-deployment usi
 
 </details>
 
+### Additional commands
+
+<details>
+_Manage integrations_
```
Comment (AlexsJones) [inline README.md:278]: For some reason this is rendering as `_Manage integrations_`

**Review by AlexsJones — APPROVED** [review]
Needs fix on `_Manage integrations_` but looks good!

**Pull request merged**